src/main/java/no/fintlabs/kafka/InstanceReceivedEventConsumerConfiguration.java

```diff
@@ -14,13 +16,26 @@
 @Configuration
 public class InstanceReceivedEventConsumerConfiguration {
 
+    private final EventTopicNameParameters formDefinitionEventTopicNameParameters;
+
+    public InstanceReceivedEventConsumerConfiguration(
+            EventTopicService eventTopicService,
+            @Value("${fint.kafka.topic.instance-retention-ms}") long retentionMs
```

Comment:
Med tanke på at det for følgende topics i denne tjenesten skal settes retention tid, bør propertien brukes til alle:
- instance-received
- instance-registered
- instance-registration-error 
- instance-requested-for-retry
- instance-retry-request-error

Foreslår at vi legger til følgende i application.yaml:
```
fint:
  flyt:
    instance-service:
      kafka:
        topic:
          instance-processing-events-retention-time-ms: 345600000
```
Foreslår å legge det under applikasjonens properties (application.yaml) og ikke sammen med properties som er definert i fint-kafka-rammeverket (application-flyt-kafka.yaml).

I denne filen blir det i så fall slik:
```suggestion
            @Value("${fint.flyt.instance-service.kafka.topic.instance-processing-events-retention-time-ms}") long retentionMs
```

Per nå så er retention time for `instance-requested-for-retry` styrt av property `fint.flyt.instance-service.time-to-keep-instance-in-days`, og den bør endres til å bruke ny foreslått property for å styre retention time. Så lenge `instance-processing-events-retention-time-ms` er mindre eller lik `fint.flyt.instance-service.time-to-keep-instance-in-days` så er det greit.